packages/core/src/libraries/quota.ts

```diff
@@ -42,18 +46,15 @@ export const createQuotaLibrary = (cloudConnection: CloudConnectionLibrary) => {
       return;
     }
 
-    const {
-      planId,
-      quota: fullQuota,
-      usage: fullUsage,
-      isEnterprisePlan,
-    } = await getTenantSubscriptionData(cloudConnection);
+    const { planId, isEnterprisePlan, quota: fullQuota } = await subscription.getSubscriptionData();
 
     // Do not block Pro/Enterprise plan from adding add-on resources.
     if (shouldReportSubscriptionUpdates(planId, isEnterprisePlan, key)) {
       return;
     }
 
+    const { usage: fullUsage } = await getTenantSubscriptionData(cloudConnection);
```

Comment:
`getTenantSubscriptionData` vs `subscription.getSubscriptionDate` is confusing, better rename the `getTenantSubscriptionData` util method.

Reply:
Updated.